ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/OperationsReOrgManager.java

```diff
@@ -123,23 +123,24 @@ private void processNonCanonicalBlockAttestations(
     // verified anymore and we need to make sure later on
     // that they're being included on the correct fork.
     attestations.forEach(
-        attestation -> {
-          attestationManager
-              .onAttestation(ValidateableAttestation.from(recentChainData.getSpec(), attestation))
-              .finish(
-                  result ->
-                      result.ifInvalid(
-                          reason ->
-                              LOG.debug(
-                                  "Rejected re-queued attestation from block: {} due to: {}",
-                                  blockRoot,
-                                  reason)),
-                  err ->
-                      LOG.error(
-                          "Failed to process re-queued attestation from block: {}",
-                          blockRoot,
-                          err));
-        });
+        attestation ->
+            attestationManager
+                .onAttestation(
+                    ValidateableAttestation.fromNonCanonicalBlock(
+                        recentChainData.getSpec(), attestation))
```

Comment:
this could become something like `ValidatableAttestation.fromVerifiedAttestation` or something...

Reply:
Personally I'd do something like `fromReorgedBlock` (or maybe `fromValidatedBlock`). Being clearer about the source makes it easier to know which properties should be set (especially if we add new properties to `ValidatableAttestation` in the future.  So attestations from a reorged block have the `signatureValidated` property set.